client/my-sites/plans/ecommerce-trial/index.tsx

```diff
@@ -0,0 +1,389 @@
+					subtitle: translate( 'Provide customers with an easy way to track their shipment.' ),
+				},
+				{
+					title: translate( 'Shipping rates' ),
+					subtitle: translate(
+						'Define multiple shipping rates based on location, price, weight, or other criteria.'
+					),
+				},
+				{
+					title: translate( 'Conditional shipping and payments' ),
+					subtitle: translate(
+						'Use conditional logic to restrict the shipping and payment options.'
+					),
+				},
+				{
+					title: translate( 'Returns and warranty' ),
+					subtitle: translate(
+						'Manage the RMA process, add warranties to products and let customers request/manage returns from their account.'
+					),
+				},
+			],
+		},
+	];
+
+	return (
+		<>
+			<BodySectionCssClass bodyClass={ [ 'is-trial-plan' ] } />
+
+			<Card className="e-commerce-trial-plans__trial-card">
+				<div className="e-commerce-trial-plans__trial-card-content">
+					<p className="e-commerce-trial-plans__card-title">
+						{ translate( 'You’re in a free trial store' ) }
+					</p>
+					<p className="e-commerce-trial-plans__card-subtitle">
+						{ isTrialExpired
+							? translate(
+									'Your free trial has expired. Sign up to a plan to unlock new features and keep your store running.'
+							  )
+							: translate(
+									'Your free trial will end in %(daysLeft)d day. Sign up to a plan by %(expirationdate)s to unlock new features and keep your store running.',
+									'Your free trial will end in %(daysLeft)d days. Sign up to a plan by %(expirationdate)s to unlock new features and keep your store running.',
+									{
+										count: eCommerceTrialDaysLeftToDisplay,
+										args: {
+											daysLeft: eCommerceTrialDaysLeftToDisplay,
+											expirationdate: readableExpirationDate,
+										},
+									}
+							  ) }
+					</p>
+				</div>
+				<div className="e-commerce-trial-plans__chart-wrapper">
+					<DoughnutChart
+						progress={ trialProgress }
+						text={ eCommerceTrialDaysLeftToDisplay?.toString() }
+					/>
+					<br />
+					<span className="e-commerce-trial-plans__chart-label">
+						{ isTrialExpired
+							? translate( 'Your free trial has expired' )
+							: translate( 'day left in trial', 'days left in trial', {
+									count: eCommerceTrialDaysLeftToDisplay,
+							  } ) }
+					</span>
+				</div>
+			</Card>
+
+			<div className="e-commerce-trial-plans__interval-toggle-wrapper">
+				<SegmentedControl compact primary={ true } className={ segmentClasses }>
+					<SegmentedControl.Item
+						selected={ interval === 'monthly' }
+						onClick={ () => setInterval( 'monthly' ) }
+					>
+						<span>{ translate( 'Pay Monthly' ) }</span>
+					</SegmentedControl.Item>
+
+					<SegmentedControl.Item
+						selected={ interval === 'yearly' }
+						onClick={ () => setInterval( 'yearly' ) }
+					>
+						<span>{ translate( 'Pay Annually' ) }</span>
+					</SegmentedControl.Item>
+				</SegmentedControl>
+			</div>
```

Comment:
The main Plans UI defaults to the current term if no term is specified in the URL, but the toggles switch to a URL that includes the term in the path, i.e. `/plans/[yearly|monthly]/[siteSlug]`, and this URL is kept when switching sites.
So I think it would be best to skip the toggle for this PR, and implement it in a follow-up, as the behaviour is just weird as it is. Alternatively, we should let the interval be injected as a prop from the plans wrapper - it looks like it is available in the `intervalType` prop in the main `Plans` component.

Also, we probably want the CTAs and flows to default to monthly for eCommerce trials, so we should cater for that too.

Reply:
I think it has complexity enough to have its own PR.